containers/ecr-viewer/src/styles/custom-uswds-styles.scss

```diff
@@ -84,3 +84,10 @@
 .short-tooltip + .usa-tooltip__body {
   min-width: min(350px, 0.25 * (100vw - 16rem));
 }
+
+.usa-prose > h5 {
```

Comment:
could you move this to just before the summary box rule? (vaguely alphabetical file)